supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesPollingActor.scala

```diff
@@ -109,13 +108,31 @@ class JesPollingActor(pollingManager: ActorRef) extends Actor with ActorLogging
     * Warning: Only use this from inside a receive method.
     */
   private def scheduleCheckForWork(): Unit = {
-    context.system.scheduler.scheduleOnce(BatchInterval) { pollingManager ! JesApiQueryManager.RequestJesPollingWork(MaxBatchSize) }
+    context.system.scheduler.scheduleOnce(batchInterval) { pollingManager ! JesApiQueryManager.RequestJesPollingWork(MaxBatchSize) }
     ()
   }
 }
 
 object JesPollingActor {
-  def props(pollingManager: ActorRef) = Props(new JesPollingActor(pollingManager))
+  def props(pollingManager: ActorRef, jesConfiguration: JesConfiguration) = Props(new JesPollingActor(pollingManager, jesConfiguration))
+
+  // The Batch API limits us to 100 at a time
+  val MaxBatchSize = 100
+
+  /**
+    * Given the Genomics API queries per 100 seconds and given MaxBatchSize will determine a batch interval which
+    * is at 90% of the quota. The (still crude) delta is to provide some room at the edges for things like new
+    * calls, etc.
+    *
+    * Forcing the minimum value to be 1 second, for now it seems unlikely to matter and it makes testing a bit
+    * easier
+    */
+  def determineBatchInterval(qps: Int): FiniteDuration = {
+    val batchesPerSecond = qps / MaxBatchSize.toDouble // Force this to be floating point in case the value is < 1
+    val maxInterval = 1 / batchesPerSecond
```

Comment:
Should we check somewhere that qps is not 0 ?
Of course it should never be 0 but if for some reason it happens it would be nice not to have a giant crash with `java.lang.ArithmeticException` :p